roles/openshift_docker_facts/tasks/main.yml

```diff
@@ -65,6 +65,6 @@
 
 - name: Set docker version to be installed
   set_fact:
-    docker_version: "{{ '1.9.1' }}"
+    docker_version: "{{ '1.10.3' }}"
```

Comment:
Do we want to mandate a specific docker version for 3.2, or would it make sense to just ensure gte 1.9.1 (at least until we have a ceiling on supported version)?